internal/cephfs/volume.go

```diff
@@ -34,6 +34,15 @@ var (
 	// Subvolume group create gets called every time the plugin loads, though it doesn't result in error
 	// its unnecessary
 	cephfsInit = false
+
+	// ceph fs subvolume resize is supported from 14.2.8 onwards
+	majorCephVersionForResize = 14
+	minorCephVersionForResize = 2
+	patchCephVersionForResize = 8
```

Comment:
This version checking is not sufficient. 15.0.x does not have the support. You'll need to do something like `kernelSupportsQuota()`, or just try to execute the new command, and fallback to the old way of resizing (like RBD does for deleting images through a task, or inline).

Reply:
are you sure that 15.0.x doesn't have support?  let me remove the complete check of ceph version based on the command availability we can do it.

Reply:
I thought it got added to Ceph 15.2.x, but I can be wrong. It is a backported feature in any case, so I think it it not in all 15.x versions.